### PR TITLE
Add mission.md

### PR DESCRIPTION
**Pull request by fabiand**
Contains the projects motivation, goals, and mission.

Signed-off-by: Fabian Deutsch <fabiand@fedoraproject.org>

### CODE REVIEW

**Comment by berrange** [comment]
Based on the discussions at the f2f meeting where we came up with new text, I've proposed this new PR https://github.com/kubevirt/kubevirt.github.io/pull/3

**Comment by fabiand** [comment]
This PR also already includes the updates from the f2f.

**Comment by berrange** [comment]
Urgh, github still claims the commit is 19 days old so it didn't look like it was updated :-(

In any case, there are some notable differences between what you've proposed & what I've proposed. In particular I've put the high level aims directly on the front page to replace the text that was there previously. I've also linked to the new page, so people know it exists & can find it :-P  What I didn't do was include the "FAQ" content yet, since I was going to consider where best to fit that in, as the "FAQ" concept tends to be thought of as somewhat anachronistic these days. I thought perhaps just a call it a 'technology comparison' page instead.

File: mission-faq.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+
+## Is KubeVirt like virtlet?
+
+Somewhat. [virtlet](https://github.com/Mirantis/virtlet) is a CRI
+implemetation to run virtual machines instead of containers.
+
+The key differences to KubeVirt are:
+
+- **It's a CRI.** This implies that the VM runtime is on the host, and that the
```
Comment (davidvossel) [inline mission-faq.md]: it's/virtlet's/

File: mission-faq.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+level.
+KubeVirt on the other hand is about allowing to manage virtual machines on a
+cluster level.
+
+But beyond that it's also how virtual machines are exposed.
+ClearContainers hide the fact that a virtual machine is used, but KubeVirt is
+highly interested in providing an API to configure a virtual machine.
+
+
+## Is KubeVirt like virtlet?
```
Comment (davidvossel) [inline mission-faq.md]: I think we should steer away from discussing the technical differences and talk more about the high level differences... So how the goals, motivation, and direction of the projects differ.  The technical details alone don't present a strong argument for why both projects should exist.

**Comment by berrange** [comment]
CLosing since we updated the mission in another PR

**Pull request closed without merging**